protobuf/v1/pool.proto

```diff
@@ -0,0 +1,58 @@
+
+syntax = "proto3";
```

Comment:
Can we define the rpc service here as well? Ideally, we would not have a mayastor.proto for v1.